crates/biome_analyze/src/rule.rs

```diff
@@ -543,6 +543,14 @@ pub trait Rule: RuleMeta + Sized {
     /// `diagnostic` or `action` on it
     fn run(ctx: &RuleContext<Self>) -> Self::Signals;
 
+    /// Returns the value associated with the given signal.
+    ///
+    /// This allows suppression of specific values for a given rule, without
+    /// suppressing other values for the same rule.
+    fn values_for_signal(_signal: &Self::State) -> Vec<String> {
```

Comment:
The concept of "value" is very broad, and I am sure I understand it. What's a "value" for a signal? I know we are passing this "value" from a suppression comment, but it's very hard to understand what they are for and how to use them.

Maybe you should implement update the contribution guide with an example, so I might be able to better understand what's this about

Reply:
As I was writing a response to this comment, I think I may have realized a way to explain it better :) Let me give it a shot!

Reply:
Renamed `value` to `instance` and updated the docs.